app/flatpak-builtins-search.c

```diff
@@ -208,9 +208,15 @@ flatpak_builtin_search (int argc, char **argv, GCancellable *cancellable, GError
       for (i = 0; i < apps->len; ++i)
         {
           AsApp *app = g_ptr_array_index (apps, i);
-          const guint score = as_app_search_matches (app, search_text);
+          guint score = as_app_search_matches (app, search_text);
           if (score == 0)
-            continue;
+            {
+              const char *app_id = as_app_get_id_filename (app);
+              if (g_strstr_len (app_id, -1, search_text) != NULL)
```

Comment:
Not important but why not just `strstr()`, probably more optimized implementation.

Reply:
Yeah I'll make it case insensitive. I think even though the app ID should never be NULL, it's nice to use the `g_` versions of things.